src/Pages/Applications/DevtronApps/Details/AppConfigurations/AppConfig.utils.ts

```diff
@@ -248,10 +249,20 @@ export const getNavItems = (
                 title: 'Environment Override',
                 href: `/app/${appId}/edit/env-override`,
                 stage: 'ENV_OVERRIDE',
-                isLocked: !isUnlocked.envOverride,
+                isLocked: !_isUnlocked.envOverride,
             },
         ]
     }
 
     return { navItems }
 }
+
+export function isCIPipelineCreated(responseArr: AppConfigStatusResponseItem[]): boolean {
+    const ciPipeline = responseArr.find((item) => item.stageName === 'CI_PIPELINE')
```

Comment:
Please add handling for null responseArr

Reply:
function has been copied pasted from other location

Reply:
discussed